db/sql/create-timeEntry-and-timePeriodType-table.sql

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
I know this ticket is for TimeEntry specifically but don't we have to also have a TimeCard entity that the TimeEntry entity is associated with?
According to the [record-time doc](https://github.com/UKHomeOffice/callisto-timecard-restapi/blob/main/docs/features/record-time/index.md):

> A TimeEntry belongs to a TimeCard with an identical TimeCardId. When creating a TimeEntry it must be associated with a TimeCard, it cannot exist on its own.

Reply:
Hi @daleelR - that version on the main branch is out of date (apologies). There's a version on the payload-model branch that needs a pull request creating to get it merged in

https://github.com/UKHomeOffice/callisto-timecard-restapi/blob/payload-model/docs/features/record-time/index.md

Reply:
Ah right thanks for the clarification @wweatherill-homeoffice. I'll resolve this comment.

Reply:
A decision was made that TimeCard need not exist for now. It has the drawback that every entity for a day now needs to duplicate ownerid and so the database isn't perfect but that's where we are for now

Reply:
Yeah makes sense, thanks David.